places/#naakheel-hometown.md

```diff
@@ -0,0 +1,5 @@
+# Muscat, Oman
+
+## Great Beaches
+## Beautiful Mountains 
+## Relaxing Corniche 
```

Comment:
What is Corniche?  Please be sure this thing actually exists before checking it in.